index.js

```diff
@@ -14,19 +14,31 @@ var extension = function(module, filename) {
 require.extensions['.handlebars'] = extension;
 require.extensions['.hbs'] = extension;
 
-var replaceText = function(catalog, opts, chunk, enc, callback) {
-  var template = _.template(chunk.toString(), {
-    interpolate: opts.interpolate || /{{trans\s"([\s\S]+?)"}}/g
-  });
-
-  var translatedString = template(catalog)
+var escapeStr = function(str) {
+  return str
     .replace(/\n/g, '\\n')
     .replace(/"/g, '\\"');
+};
+
+var replaceText = function(catalog, opts, chunk, enc, callback) {
+  var template = chunk.toString();
+  var re = opts.interpolate || /\{\{trans\s*(?:"([^"]+)"|\'([^\']+)\')\s*\}\}/g;
+  var chunkString, match, msgid, needle, translated;
+
+  translated = template;
```

Comment:
nevermind, i just saw `Regex.prototype.exec()`'s successive match feature. that's dope